### ISSUE TITLE
Fallback values reported as box-shadows

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#381]
```css
box-shadow: var(--tw-ring-offset-shadow, 0 0 #0000), var(--tw-ring-shadow, 0 0 #0000), var(--tw-shadow);
```

is reported as 2x `0 0 #0000`

<img width="1119" alt="Screenshot 2024-01-12 at 08 59 29" src="https://github.com/projectwallace/css-analyzer/assets/1536852/a72024d6-6282-45e4-b823-8e8ea5bbfd5e">
